docker/register.sh

```diff
@@ -174,7 +198,7 @@ setup_operator() {
         echo "Error: Failed to register operator $public_key to operator sets"
         exit 1
     fi
```

Comment:
This entire section of submit -> registerForOperatorSets needs to be ignored if the user is already in the operator set and / or they already have LST

Also `register_operator_with_avs` was throwing to dev/null and was impossible to debug without removing. we should really show as much as possible to the user
)side note: register_operator_with_avs failed bc it was trying to read from a file the operator did not have access too, only the deployer)